src/settings/MappingProfiles/detailsSections/view/InvoiceDetailsSection/ExtendedInformation.js

```diff
@@ -0,0 +1,113 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+
+import {
+  Accordion,
+  Col,
+  Row,
+  KeyValue,
+  NoValue,
+} from '@folio/stripes/components';
+
+import { ProhibitionIcon } from '../../../../../components';
+
+import { TRANSLATION_ID_PREFIX } from '../../constants';
+import {
+  getFieldValue,
+  renderCheckbox,
+} from '../../utils';
+import { mappingProfileFieldShape } from '../../../../../utils';
+
+export const ExtendedInformation = ({ mappingDetails }) => {
+  const noValueElement = <NoValue />;
+  const prohibitionIconElement = fieldName => <ProhibitionIcon fieldName={fieldName} />;
+
+  const folioInvoiceNumberValue = getFieldValue(mappingDetails, 'folioInvoiceNo', 'value');
+  const paymentMethodValue = getFieldValue(mappingDetails, 'paymentMethod', 'value');
+  const checkSubscriptionOverlapValue = getFieldValue(mappingDetails, 'chkSubscriptionOverlap', 'booleanFieldAction');
+  const exportToAccountingValue = getFieldValue(mappingDetails, 'exportToAccounting', 'booleanFieldAction');
+  const currencyValue = getFieldValue(mappingDetails, 'currency', 'value');
+  const currentExchangeRateValue = getFieldValue(mappingDetails, 'currentExchangeRate', 'value');
+  const setExchangeRateValue = getFieldValue(mappingDetails, 'exchangeRate', 'value');
+
+  const checkSubscriptionOverlap = renderCheckbox('invoice.extendedInformation.field.checkSubscriptionOverlap', checkSubscriptionOverlapValue);
+  const exportToAccounting = renderCheckbox('invoice.extendedInformation.field.exportToAccounting', exportToAccountingValue);
+
+  return (
+    <Accordion
+      id="extended-information"
+      label={<FormattedMessage id={`${TRANSLATION_ID_PREFIX}.invoice.extendedInformation.section`} />}
+    >
+      <Row left="xs">
+        <Col
+          data-test-invoice-number
+          xs={3}
+        >
+          <KeyValue
+            label={<FormattedMessage id={`${TRANSLATION_ID_PREFIX}.invoice.extendedInformation.field.folioInvoiceNumber`} />}
+            value={folioInvoiceNumberValue || prohibitionIconElement('folio-invoice-number')}
+          />
+        </Col>
+        <Col
+          data-test-payment-method
+          xs={3}
+        >
+          <KeyValue
+            label={<FormattedMessage id={`${TRANSLATION_ID_PREFIX}.invoice.extendedInformation.field.paymentMethod`} />}
+            value={paymentMethodValue || noValueElement}
```

Comment:
The `KeyValue` component checks whether the value exists or not and if not, then renders the `NoValue` component on its own, so you do not need this check.
```suggestion
            value={paymentMethodValue}
```

Reply:
done